libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Plutus/Examples.hs

```diff
@@ -258,9 +774,92 @@ evenRedeemer2 =
     . PlutusBinary
     . pack
     . ( \case
-          SPlutusV1 -> [88, 25, 1, 0, 0, 34, 83, 51, 87, 52, 102, 225, 210, 0, 3, 55, 12, 110, 180, 0, 146, 0, 65, 18, 0, 17, 97]
-          SPlutusV2 -> [88, 25, 1, 0, 0, 34, 83, 51, 87, 52, 102, 225, 210, 0, 3, 55, 12, 110, 180, 0, 146, 0, 65, 18, 0, 17, 97]
-          SPlutusV3 -> [88, 24, 1, 1, 0, 34, 83, 51, 87, 52, 102, 225, 210, 0, 3, 55, 12, 110, 180, 0, 146, 0, 65, 128, 0, 177]
+          SPlutusV1 ->
+            [ 88
+            , 25
+            , 1
+            , 0
+            , 0
+            , 34
+            , 83
+            , 51
+            , 87
+            , 52
+            , 102
+            , 225
+            , 210
+            , 0
+            , 3
+            , 55
+            , 12
+            , 110
+            , 180
+            , 0
+            , 146
+            , 0
+            , 65
+            , 18
+            , 0
+            , 17
+            , 97
+            ]
+          SPlutusV2 ->
+            [ 88
+            , 25
+            , 1
+            , 0
+            , 0
+            , 34
+            , 83
+            , 51
+            , 87
+            , 52
+            , 102
+            , 225
+            , 210
+            , 0
+            , 3
+            , 55
+            , 12
+            , 110
+            , 180
+            , 0
+            , 146
+            , 0
+            , 65
+            , 18
+            , 0
+            , 17
+            , 97
+            ]
+          SPlutusV3 ->
+            [ 88
+            , 24
+            , 1
+            , 1
+            , 0
+            , 34
+            , 83
+            , 51
+            , 87
+            , 52
+            , 102
+            , 225
+            , 210
+            , 0
+            , 3
+            , 55
+            , 12
+            , 110
+            , 180
+            , 0
+            , 146
+            , 0
+            , 65
+            , 128
+            , 0
+            , 177
+            ]
```

Comment:
I'm not sure I'm liking this... :confounded:

Reply:
Haha, I agree it looks a little awkward!  We could presumably split them in two lists and concatenate them but it's a strange thing to come across. I'd say leave them like this, if they bother us we can come up with something.

Reply:
This code is generated by a preprocessor, so I don't think it needs to be pretty or anything.
That being said if we want to make it more readable we could switch to decoding a hex string, rather than packing a list of 8 bit words. This would be more concise.